### ISSUE TITLE
Extreme values

### ISSUE THREAD

**Issue by neurolabusc** [vsoch/TtoZ#3]
Your project spurred a response [from the AFNI team](https://github.com/afni/afni/issues/301). One needs to be careful generating histograms to ensure your range and number of bins accurately captures the data. Thanks for bringing this interesting paper to my attention. We often neglect to discuss the handling of extreme values. It is clear that AFNI clips values that exceed z > 13.

**Comment by vsoch** [comment]
Wow that's great! We found this issue a long time ago, and dug up that old paper with the fix and I implemented it here. I'm glad that it's been useful even after all these years! if you want to cite/reference the repository anywhere, there is a DOI associated with the repository:

https://zenodo.org/record/32508#.YTOsoFtMFH4

And of course Hughett's paper!

https://github.com/vsoch/TtoZ/blob/master/doc/JStats_Hughett.pdf

ping @jokedurnez and @chrisgorgo who were in my lab at the time and I think likely involved with working on this and would be interested to follow / participate in the discussion.

Thanks for the ping!
